packages/core/src/V2/nodes/console.test.ts

```diff
@@ -0,0 +1,37 @@
+import { describe, expect, vi } from 'vitest';
+import { LinkNodePorts } from './link';
+import { firstValueFrom, interval, of, lastValueFrom } from 'rxjs';
+import { Console } from './console';
+import { take } from 'rxjs/operators';
+
+describe('Console', () => {
```

Comment:
* We could consider instances/variables as lowercase always
* Historically, I just gave the ComputerConfigs capitalization based on their "importance" / central nature. But not sure that is a clean idea
* Files should always match case of primary export.

WDYT? Lets get a convention here

Reply:
1. I agree with that point!
2. Except for React Components, the rest could follow the first advice.
3. The JS file name is part of the module identifier, so I believe it should adhere to a uniform standard rather than changing according to the content inside the file.
Maybe we can refer to Google's JS guide, what do you think? https://google.github.io/styleguide/jsguide.html#file-name